integration_tests/dbt_project.yml

```diff
@@ -13,6 +13,7 @@ log-path: "logs"
 
 vars:
   tasman_dbt_package:
```

Comment:
NB: Is the name of the dbt project that makes this package matter for publishing to dbt Hub? Line 1 on the `dbt_project.yml` defines `name: 'tasman_dbt_package'` -- I reckon this could be more representative (i.e. `tasman_identity_graph`? 
It would make setting this variable much more explicit and clear in an actual dbt project setup.

Reply:
I agree - good spot!